src/implementations/token-registry/issue.test.ts

```diff
@@ -1,66 +1,49 @@
-import { TradeTrustErc721Factory } from "@govtechsg/token-registry";
+import { TradeTrustToken__factory } from "@govtechsg/token-registry/contracts";
 import { Wallet } from "ethers";
-import { join } from "path";
+
 import { TokenRegistryIssueCommand } from "../../commands/token-registry/token-registry-command.type";
 import { addAddressPrefix } from "../../utils";
 import { issueToTokenRegistry } from "./issue";
 
-jest.mock("@govtechsg/token-registry");
+jest.mock("@govtechsg/token-registry/contracts");
 
 const deployParams: TokenRegistryIssueCommand = {
-  to: "0xabcd",
+  beneficiary: "0xabcd",
+  holder: "0xabce",
   tokenId: "0xzyxw",
   address: "0x1234",
   network: "goerli",
-  gasPriceScale: 1,
   dryRun: false,
 };
 
-// TODO the following test is very fragile and might break on every interface change of TradeTrustErc721Factory
-// ideally must setup ganache, and run the function over it
 describe("token-registry", () => {
   describe("issue", () => {
     jest.setTimeout(30000);
-    const mockedTradeTrustErc721Factory: jest.Mock<TradeTrustErc721Factory> = TradeTrustErc721Factory as any;
+    const mockedTradeTrustTokenFactory: jest.Mock<TradeTrustToken__factory> = TradeTrustToken__factory as any;
     // eslint-disable-next-line @typescript-eslint/ban-ts-comment
     // @ts-ignore mock static method
-    const mockedConnect: jest.Mock = mockedTradeTrustErc721Factory.connect;
+    const mockedConnectERC721: jest.Mock = mockedTradeTrustTokenFactory.connect;
     const mockedIssue = jest.fn();
     const mockCallStaticSafeMint = jest.fn().mockResolvedValue(undefined);
-    beforeEach(() => {
-      delete process.env.OA_PRIVATE_KEY;
-      mockedTradeTrustErc721Factory.mockReset();
-      mockedConnect.mockReset();
-      mockCallStaticSafeMint.mockClear();
-      mockedConnect.mockReturnValue({
-        "safeMint(address,uint256)": mockedIssue,
-        callStatic: {
-          "safeMint(address,uint256)": mockCallStaticSafeMint,
-        },
-      });
-      mockedIssue.mockReturnValue({
-        hash: "hash",
-        wait: () => Promise.resolve({ transactionHash: "transactionHash" }),
-      });
-    });
-
-    it("should take in the key from environment variable", async () => {
-      process.env.OA_PRIVATE_KEY = "0000000000000000000000000000000000000000000000000000000000000002";
 
-      await issueToTokenRegistry(deployParams);
-
-      const passedSigner: Wallet = mockedConnect.mock.calls[0][1];
-      expect(passedSigner.privateKey).toBe(`0x${process.env.OA_PRIVATE_KEY}`);
+    mockedIssue.mockReturnValue({
+      hash: "hash",
+      wait: () => Promise.resolve({ transactionHash: "transactionHash" }),
     });
 
-    it("should take in the key from key file", async () => {
-      await issueToTokenRegistry({
-        ...deployParams,
-        keyFile: join(__dirname, "..", "..", "..", "examples", "sample-key"),
-      });
+    const mockTTERC721Contract = {
+      mint: mockedIssue,
```

Comment:
(nitpick): the casing caught my eye, thinking of camelcase regardless of acronym / abbreviation. ok to leave as this too